oshi-core/src/main/java/oshi/hardware/common/AbstractNetworkIF.java

```diff
@@ -68,8 +69,7 @@
      * Construct a {@link NetworkIF} object backed by the specified
      * {@link NetworkInterface}.
      *
-     * @param netint
-     *            The core java {@link NetworkInterface} backing this object.
+     * @param netint The core java {@link NetworkInterface} backing this object.
```

Comment:
Not that big a deal, but we do use consistent formatting to avoid whitespace/format noise in the diffs.  If possible, I'd prefer you set up your IDE to use the same format settings.  You can find them [here](https://github.com/oshi/oshi/tree/master/oshi-dist/src/main/resources/eclipse) if you use Eclipse, and I think IntelliJ can import eclipse settings (basically the "Eclipse Default" with spaces instead of tabs.)

If you can't easily format, don't worry, I'll just run a source format before merging.